lib/lolcommits/platform.rb

```diff
@@ -1,68 +1,63 @@
 # -*- encoding : utf-8 -*-
 require 'mini_magick'
+require 'rbconfig'
 
 module Lolcommits
   class Platform
     # A convenience name for the platform.  Mainly used for metaprogramming.
     # @return String
     def self.platform
-      if platform_fakeplatform?   then ENV['LOLCOMMITS_FAKEPLATFORM']
-      elsif platform_fakecapture? then 'Fake'
-      elsif platform_mac?         then 'Mac'
-      elsif platform_linux?       then 'Linux'
-      elsif platform_windows?     then 'Windows'
-      elsif platform_cygwin?      then 'Cygwin'
+      if platform_mac?        then 'Mac'
+      elsif platform_linux?   then 'Linux'
+      elsif platform_windows? then 'Windows'
+      elsif platform_cygwin?  then 'Cygwin'
       else
         fail 'Unknown / Unsupported Platform.'
       end
     end
 
+    # The capturer class constant to use
+    # @return Class
+    def self.capturer_class(animate = false)
+      Object.const_get('Lolcommits').const_get(
+        ENV['LOLCOMMITS_CAPTURER'] || "Capture#{platform}#{animate ? 'Animated' : nil}"
```

Comment:
Yep, I think thats a good idea too.  We don't get much benefit from this right now, except maybe adding the `Animated` part.  I'll change this to not use the `const_get` and see how it looks.